site/content/community/guides.en.md

```diff
@@ -17,6 +18,10 @@ Share your applications, videos, and blog posts with fellow Copilots!
 
 | Title      | Description                          |
 | ----------- | ------------------------------------ |
+| [**Container Day: Amazon ECS Edition-- Demo: deploy a modern application on AWS Fargate with ECS Copilot**](https://www.youtube.com/watch?v=k01AdJC-dcA&t=14236s) by <a href="https://twitter.com/efekarakus">@efekarakus</a> and <a href="https://twitter.com/realadamjkeller">@realadamjkeller</a> | Efe goes beyond the basics, demoing how to grow [an application](https://github.com/efekarakus/day2-with-copilot); he highlights the `alias` field, secrets and environment variables, addons, and more. |
+| [**Copilot– What AWS ECS and Fargate Container Management Should Have Been**](https://www.youtube.com/watch?v=YCCFK2RRm7U) by <a href="https://twitter.com/vfarcic">@vfarcic</a> | Viktor lauds Copilot for its simplicity and ease of use in this clear, step-by-step demo. |
```

Comment:
It seems like we have more videos on "Containers from the couch" channel. Do we want to include them as well?

Reply:
They're all lumped together as a series in the videos list...but it might be nice to list them separately, especially since they cover different topics.

Reply:
I listed out and linked to specific episodes in a separate commit.